decidim/decidim
# Redesign: admin screen overflow on smaller screens

#### :tophat: What? Why?
When adjusting the screen size while being logged in in the admin section section of Decidim on smaller screens, you notice the full width of the layout in certain modules was not being adjusted properly. 

While inside in **Processes**, this affected the **Meetings**, **Accountability** and **Proposals** components.

This issue also fixes a breadcrumb issue affecting user options on some modules affected. A trigger button has now been added to display the menu when viewing on a smaller screen. See images below. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11786

#### Testing
1. Login as an Admin
2. Head over to edit and click Processes
3. Find one and click one and you see a no. of components on the left. 
4. Click **Accountability**, and adjust your screen size to the following dimensions: 1024x768 while being on a horizontal scroll and 125% zoom. 
5. See the improvement.

### :camera: Screenshots

Before:
<img width="898" alt="Screenshot 2023-11-06 at 10 24 01" src="https://github.com/decidim/decidim/assets/101816158/daf542cb-b29b-4064-b3b5-339ab7f87329">

After:
<img width="950" alt="Screenshot 2023-11-06 at 10 25 50" src="https://github.com/decidim/decidim/assets/101816158/718d73e4-affa-434b-8362-88d7b48c69a2">

Menu button added: 

<img width="781" alt="Screenshot 2023-11-13 at 13 54 46" src="https://github.com/decidim/decidim/assets/101816158/a3865bd3-7830-421e-ba2d-16a82bcee343">

Menu after click: 

<img width="798" alt="Screenshot 2023-11-13 at 13 54 51" src="https://github.com/decidim/decidim/assets/101816158/0f87de16-bce4-4353-adbb-0780b41bed19">


:hearts: Thank you!
